### DISCUSSION TITLE
[Resolved] the "settings" page not work for me in vercel dashboard

### DISCUSSION THREAD

**Discussion by the-research** [vercel/community#2475] [Bug Report]
the settings page in vercel dashboard not showing me the form (I see a loader)
when I open the console it shows me 500 server error on this route: (GET https://vercel.com/api/v1/frameworks 500 SERVER ERROR)

is anyone here discover the same issue?

**Comment by amyegan** [comment]
Thanks for reporting this, @the-research. The team is investigating.

**Reply by the-research** [reply]
Do you need my account details?

**Reply by amyegan** [reply]
The team was able to locate the source of the problem using our logs. The API problem is resolved now, and the dashboard is working normally.

**Reply by the-research** [reply]
Thanks! Working

**Comment by CarlessoMauricio** [comment]
Same here, how fix?
